yt/visualization/eps_writer.py

```diff
@@ -1364,16 +1368,16 @@ def return_cmap(cmap=None, label="", range=(0,1), log=False):
         name of the (matplotlib) colormap to use
     label : string
         colorbar label
-    range : tuple of floats
+    crange : tuple of floats
```

Comment:
I know this is nitpicky, but I think the `range` of a colormap makes more sense as a kwarg than the `crange` of a colormap. I think these kwargs should be changed back to range.

Reply:
Ok. The idea was to avoid using python keywords as variable names but in any case it's out of scope so I'm happy to revert this change.